BraveUI/Then.swift

```diff
@@ -0,0 +1,85 @@
+// The MIT License (MIT)
```

Comment:
Instead of copying the file to each framework, is it possible that it would still work if we just added it to the target like:

![image](https://user-images.githubusercontent.com/1530031/77559496-a58d4180-6e92-11ea-9b73-6ba8f6a4ec41.png)

Reply:
I think I tried this first but got duplicate symbol issues (since the frameworks import each other)